recipes/zimg/all/conanfile.py

```diff
@@ -0,0 +1,136 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+
+
+class ZimgConan(ConanFile):
+    name = "zimg"
+    description = "Scaling, colorspace conversion, and dithering library"
+    topics = ("conan", "zimg", "image", "manipulation")
+    homepage = "https://github.com/sekrit-twc/zimg"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "WTFPL"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    exports_sources = "patches/**"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if not self.settings.build_type:
+            self.settings.build_type = "Release"
```

Comment:
```suggestion
```

I would remove these two lines. If we are building this package we need to provide the `build_type`, we cannot assign it to a value.